cypher/cypher-docs/src/docs/dev/ql/administration/security/deny-all-graph-privileges-syntax.asciidoc

```diff
@@ -1,7 +0,0 @@
-.Command syntax
```

Comment:
Content was moved to cypherManual/asciidoc/access-control/privileges-write.adoc

https://neo4j.com/docs/cypher-manual/4.3/access-control/privileges-writes/#access-control-privileges-writes-all